core/templates/dev/head/pages/error/Error.js

```diff
@@ -17,12 +17,26 @@
  */
 
 oppia.controller('Error', [
-  '$scope', 'UrlInterpolationService',
+  '$rootScope', '$scope', 'UrlInterpolationService',
   function(
-      $scope, UrlInterpolationService) {
+      $rootScope, $scope, UrlInterpolationService) {
     $scope.oopsMintImgUrl = UrlInterpolationService.getStaticImageUrl(
       '/general/oops_mint.png');
 
     $scope.statusCode = GLOBALS.status_code;
+    switch ($scope.statusCode) {
+      case 400:
+        $rootScope.title = 'I18N_ERROR_PAGE_TITLE_400';
```

Comment:
Can we do this without populating rootscope?
I mean maybe you can use Base controller (from base.js).

ALos, you can take a look at #3969 to see how this data flows from backend to frontend!

Reply:
Do you have any ideas for how it might work without populating rootscope? The problem is that `maintitle` is bound to rootscope, so I don't think it is possible to do this without major changes to everything else(see [comment](https://github.com/oppia/oppia/blob/994ac3364f10649b8522479bfefbf70da3c999a2/core/templates/dev/head/pages/base.html#L64)).

Sorry if it's actually a lot more trivial than I think, I'm not very experienced with angular.

Reply:
Fixed